eq-author/src/components/ContentPickerv2/index.test.js

```diff
@@ -9,19 +9,27 @@ import {
 
 import ContentPicker from "./";
 
-import {
-  destinationKey,
-  EndOfQuestionnaire,
-  NextPage,
-} from "constants/destinations";
+import { EndOfQuestionnaire, NextPage } from "constants/destinations";
+import { destinationKey } from "constants/destinationKey";
+
+import { useQuestionnaire } from "components/QuestionnaireContext";
+
+jest.mock("components/QuestionnaireContext", () => ({
+  useQuestionnaire: jest.fn(),
+}));
+
+jest.mock("constants/destinations", () => ({
+  logicalDestinations: jest.fn(),
```

Comment:
don't need this logicalDesitantion mock as you've defined it in data and tests still work without it